applications/sink/rsocket-sink/README.adoc

```diff
@@ -8,10 +8,6 @@ RSocket sink to send data using RSocket protocols' fire and forget strategy.
 The **$$rsocket$$** $$sink$$ has the following options:
 
 //tag::configuration-properties[]
-$$rsocket.consumer.host$$:: $$RSocket host.$$ *($$String$$, default: `$$localhost$$`)*
-$$rsocket.consumer.port$$:: $$RSocket port.$$ *($$Integer$$, default: `$$7000$$`)*
-$$rsocket.consumer.route$$:: $$Route used for RSocket.$$ *($$String$$, default: `$$<none>$$`)*
-$$rsocket.consumer.uri$$:: $$URI that can be used for websocket based transport.$$ *($$URI$$, default: `$$<none>$$`)*
```

Comment:
Is it really correct to remove this props?
What happened? Why don't you remove similar then for all other READMEs?

Reply:
No. I needed to re-run the build after adding back in the file that was accidentally removed (https://github.com/spring-cloud/stream-applications/pull/464#discussion_r1218388487).

I have the changes locally and forgot to push the subsequent commit (doing that now).